### ISSUE TITLE
when set window.options.content, el.style is undefined

### ISSUE THREAD

**Issue by jaewon1219** [angular-ui/angular-google-maps#1716]
http://plnkr.co/edit/KnEM1D7RfJgiyLfu3dN4?p=preview

you can see console
Uncaught TypeError: Cannot set property 'opacity' of undefined

**Comment by nmccready** [comment]
Yeah there are two problems there, one the content call in [window-child-model](https://github.com/angular-ui/angular-google-maps/blob/master/src/coffee/directives/api/models/child/window-child-model.coffee#L88) at one point was passed as an object. This hack was to deal with a problem with window rendering in Chrome. But I don't think that error is something to worry about any longer. So the easiest fix is just remove this listener and ChromeFixes entirely.

**Comment by AlfonsoMonroyIV** [comment]
Mm so...where can I find this file? I don't have these path in my project..

 bower_components\angular-google-maps

Is that code compiled?..sorry Im not sure how does it work..
